plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/custom/util/SchemaKeyEditorDialog.java

```diff
@@ -261,6 +272,19 @@ public void modifyText(ModifyEvent arg0) {
                     generateSchema.setVisible(false);
                     link.setVisible(false);
                     schemaKeyTextField.setVisible(false);
+                    lblDelimiterLabel.setVisible(false);
+                    delimiterTextField.setVisible(false);
+                    grpPropertyKey.redraw();
+                } else if(FileType.CSV.toString().equals(schemaTypeCombo.getText())) {
+                	lblConnector.setVisible(false);
```

Comment:
Shall we correct small formatting issues?

Reply:
+1 Fixed